apps/damap-frontend/src/themes/custom-theme.scss

```diff
@@ -169,6 +179,11 @@ html {
   --mat-icon-color: darkorange;
 }
 
+.material-icon-white-background-gray-color {
```

Comment:
This class is not needed, since these variables are the stepper default anyways.

Reply:
The class is needed because in order to personalize the icons in the stepper, the default behavior and variables of the step cannot remain as they are. 

provide: STEPPER_GLOBAL_OPTIONS,
useValue: { displayDefaultIndicatorType: false },
![image](https://github.com/user-attachments/assets/fe9a6d18-4069-4403-82b0-de75255c59a8)

Reply:
Try deleting the class, you will observe that nothing changes. Look at the class closely, you are assigning the default stepper color values to the icon and background. Also now that i think about it, isnt the background gray and the icon white? Somethings not right here 😅
I think that `useValue: { displayDefaultIndicatorType: false },` just prevents the numerical icons from showing, the default colors still stay.